misk-hibernate/src/main/kotlin/misk/hibernate/migrate/BulkShardMigrator.kt

```diff
@@ -417,6 +465,10 @@ class BulkShardMigrator<R : DbRoot<R>, C : DbChild<R, C>> private constructor(
       insert.setObject(parameterIndex, valueMapper.apply(value))
       return 1
     }
+
+    override fun bindUpdate(update: PreparedStatement, parameterIndex: Int): Int {
+      throw UnsupportedOperationException("Cannot apply updates using SetMappingMutation")
```

Comment:
I guess this is sort of doable just pretty hard :-) Should the error message say that we haven't implemented this because it's hard? :-) You could imagine you pull up the values with a select before or you specify a SQL function as well.

Reply:
I wanted to avoid doing a select to update the entities as those are exactly the queries that are impacting more (in volume) TiDB. I think also the Jdbc ```PreparedStatement``` doesn't allow to set a value based on the existing value of a column.

Reply:
Yeah I totally wouldn't do a select! We can hopefully avoid implementing this. :-) If we do want to implement it I think we should force the user to specify a SQL function expression.